symbols/overlay29.yml

```diff
@@ -970,6 +1141,24 @@ overlay29:
         r0: entity pointer
         r1: type ID
         return: bool
+    - name: CanSeeInvisibleMonsters
+      address:
+        EU: 0x2302918
+        NA: 0x2301EEC
+      description: |-
+        Returns whether a certain monster can see other invisible monsters. To be precise, this function returns true if the monster is holding Goggle Specs or if it has the status status::STATUS_EYEDROPS.
+        
+        r0: Entity pointer
+        return: True if the monster can see invisible monsters.
+    - name: HasDropeyeStatus
+      address:
+        EU: 0x230297C
+        NA: 0x2301F50
+      description: |-
+        Returns whether a certain monster is under the effect of the status::STATUS_DROPEYE status.
```

Comment:
"the status::STATUS_DROPEYE status" -> "status::STATUS_DROPEYE" for conciseness

Reply:
:+1: